backend/app/config.py

```
@@ -6,30 +6,37 @@ class Config:
     SQLALCHEMY_TRACK_MODIFICATIONS = False
 
     def __init__(self):
-        self.SECRET_KEY = os.environ.get("SECRET_KEY")
-        if not self.SECRET_KEY:
-            if os.environ.get("APP_ENV") == "production":
-                raise ValueError("SECRET_KEY environment variable is required in production")
-            self.SECRET_KEY = "dev_secret_key"
+        self.SECRET_KEY = os.environ.get("SECRET_KEY", "dev_secret_key")
```

Pylint (3.3.7)

[convention] Attribute name "SECRET_KEY" doesn't conform to snake_case naming style

(C0103)